brat/Brat/Checker/SolvePatterns.hs

```diff
@@ -266,9 +279,23 @@ unifyNum (NumValue lup lgro) (NumValue rup rgro)
   --   2^k * x
   -- = 2^k * (y + 1)
   -- = 2^k + 2^k * y
-  demandSucc (StrictMono k (Linear (VPar (ExEnd out)))) = do
-    y <- mkPred out
+  demandSucc _sm@(StrictMono _k (Linear (VPar (ExEnd _x)))) = error "Todo..." {-do
```

Comment:
I think the whole of `demandSucc` has no [test coverage](https://github.com/CQCL/brat/actions/runs/12122152161)